source/server/connection_handler_impl.h

```diff
@@ -332,12 +346,20 @@ class ConnectionHandlerImpl : public Network::ConnectionHandler,
     std::unique_ptr<StreamInfo::StreamInfo> stream_info_;
     bool connected_{false};
   };
-
   using ActiveTcpListenerOptRef = absl::optional<std::reference_wrapper<ActiveTcpListener>>;
-  using UdpListenerCallbacksOptRef =
-      absl::optional<std::reference_wrapper<Network::UdpListenerCallbacks>>;
 
-  struct ActiveListenerDetails {
+private:
+  struct ActiveTcpListenerDetail {
```

Comment:
I don't think these two base classes are used (at least in this PR). Delete?

Reply:
Right... deleting